packages/common/auth/src/CrossmintAuth.ts

```diff
@@ -16,34 +21,68 @@ export class CrossmintAuth {
                 },
             },
         });
+        this.refreshRoute = options.refreshRoute ?? null;
     }
 
-    public static from(crossmint: Crossmint): CrossmintAuth {
-        return new CrossmintAuth(crossmint);
+    public static from(crossmint: Crossmint, options: CrossmintAuthOptions = {}): CrossmintAuth {
+        return new CrossmintAuth(crossmint, options);
     }
 
     public getJwksUri() {
         return `${this.apiClient.baseUrl}/.well-known/jwks.json`;
     }
 
-    protected async refreshAuthMaterial(refreshToken: string): Promise<AuthMaterialWithUser> {
+    protected async refreshAuthMaterial(refreshToken?: string): Promise<AuthMaterialWithUser> {
```

Comment:
Why would we allow custom refresh routes to work without a refresh token?

Reply:
When HttpOnly is set, the client can't send a refresh token because it can't access the cookie, so it needs to be fetched from the cookie in the server